### ISSUE TITLE
Request: Recommend rather than mandate tools to use for ticketing, CFP, and scheduling

### ISSUE THREAD

**Issue by blixtra** [cncf/kubernetes-community-days#184]
Currently it looks like the CNCF has chosen to take the heavy-handed approach to mandating that one use Eventbrite for ticketing, Sched for scheduling and a ticket is in the pipeline for mandating Sessionize for the CFP for all KCD.

As background, I recall a long thread "Thoughts on KubeCon" about how KubeCon was getting too big and that the CNCF should support more grassroots efforts. In fact, from this conversation we came up with the idea for [Cloud Native Rejekts](https://cloud-native.rejekts.io/). But instead of supporting efforts like that, it seems that the suggestion to do a DevOpsDays model was chosen. But the CNCF chose to go further than DevOpsDays in mandating certain tools, tools which may not work well for some for technical or philosophical reasons.

As an example of a technical reason, at Kinvolk over the last five years of doing events, we've stabilized on using [Pretalx](https://pretalx.com/p/about/) for CFP and scheduling not only because they are great tools for those purposes but because it also directly links into the live video streaming system that we run (yes, we're very DIY here), and the info beamer system running on Raspberry Pis connected to projectors and TVs around the venue. The info beamer automatically displays the schedule, next talks and picks up the twitter feed. When there is a last minute change in the schedule, we make the change in PreTalx that is immediately reflected in all these systems. This is all built on open source technologies built for the [Chaos Computer Congress](https://www.ccc.de/en/) and refined over the last 20 years.

I'd propose that for KCD, the CNCF lay down some general guidelines (CoC, local organizers, etc.) and suggestions (tools that have worked and others have used) and let the community choose where to go from there. The way it is make the CNCF seem like it wants to dictate rather than facilitate. I'd hope the latter would be the case.

**Comment by dankohn** [comment]
Thanks for the message and for the short phone call we just had.

As I explained on the phone, we're open to expanding the number of supported systems after KCDs are up and running (e.g., in 2021). At launch, we want to ensure that KCDs use a set of software that CNCF is familiar with and can help support, and that enables KCD organizers from different cities to support each other. Additionally, we understand the privacy policies of the tools we're using, and we're able to ensure that all organizers are on the same footing.

We acknowledge that this makes being a KCD organizer more difficult for employees of companies like Kinvolk, where you've already standardized on other tools. And, in China and other areas where our standard tools are blocked, we'll, of course, need to work on alternatives.

**Comment by blixtra** [comment]
I'm fine with this being closed as `wont-fix`, but I think it important to document the express disagreement about decisions, decisions that I hope will be reconsidered in the future.

For future reference, what I think would be great to have are...
* Move to a guideline system, rather than a mandate system 
* Recommendations on tools to use. I don't think there needs to be one set of recommendations. I think this should come from the community and will often be locality specific.
* A list of supported tools that organizers can choice to use, if they feel they need the support.
* The ability to use more local (EU vs US, for example) services. _Tools being blocked should not be the only criteria. Being more locally sustainable should also play a part._
* The ability to use OSS tools that don't require an outside service at all

Basically, in technical terms, I'd like to see the KCD be more an interface than an implementation.

But I do look forward to the many great local events this program has the potential to create. And it's great to have a support system that enable more folks to participate. But a more hands-off approach would be very welcome.

**Comment by dankohn** [comment]
I really appreciate the *interface not implementation* phrasing and I agree that's what we should aim to move toward once we have the confidence of an initial round of events under our belt.

Closing now, but we're reopening in the second half of 2020.
